gravityflow/gravityflow
# Fix Conditional Routing on Workflow Notification

## Description
This closes [gravityflow/backlog#154](https://github.com/gravityflow/backlog/issues/154).

This fixes an issue 'Configure Routing' under 'Workflow Notification'.

## Testing instructions
1. Create a Gravity Flow Notification step
2. Enable 'Workflow Notification'
3. Select 'Configure Routing'
4. Confirm the 'Routing' section is empty.
5. Switch to this branch, repeat steps 1-3.
6. Confirm the 'Routing' now displays fine.
7. Create some routing rules, and test if they work fine.
8. Confirm conditional routing still displays fine on Gravity Forms 2.4.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes?
N/A

## Checklist:
